circle.yml

```diff
@@ -8,6 +8,8 @@ machine:
 deployment:
   staging:
     branch: master
+    environment:
+      DEPLOY_ENV: staging
```

Comment:
From what I understand from docs, setting here will make `DEPLOY_ENV` available inside of `yarn deploy` and `yarn sentry`, yes?

Reply:
I don't know if this is a "step" or a "job" in CircleCI-speak, but yes I _think_ so.

https://circleci.com/docs/2.0/env-vars/#declaring-an-environment-variable-for-a-step